caproto/_broadcaster.py

```diff
@@ -39,8 +40,10 @@ def __init__(self, our_role, protocol_version=DEFAULT_PROTOCOL_VERSION):
         self.our_role = our_role
         if our_role is CLIENT:
             self.their_role = SERVER
+            self.role = 'SERVER'
```

Comment:
Instead of defining additional state, wherever you need `'SERVER'` you can use `repr(self.their_role)`. The `SERVER` and `CLIENT` variables are special classes that have the `__repr__` `'SERVER'` and `'CLIENT'`.

Reply:
OK